testsuite/MDAnalysisTests/coordinates/test_rdkit.py

```diff
@@ -239,6 +239,11 @@ def test_warn_no_hydrogen(self):
             match="No hydrogen atom could be found in the topology"
         ):
             u.atoms.convert_to("RDKIT")
+        rdkit_converter = mda._CONVERTERS["RDKIT"]().convert
```

Comment:
Whilst I appreciate concise code, I personally prefer that tests like these be split up for readability. If that's not feasible, then can you at least add a comment to detail that you are now testing a different condition?